configs/components/fesom/fesom-2.0.yaml

```diff
@@ -110,7 +110,16 @@ namelists:
         - namelist.oce
         - namelist.ice
         - namelist.io
+
+# FIXME(KH, PG): This will need to be put in correctly.
+# NOTE(PG): For awiesm test on levante --> cvmix namelist back in, recom namelist out!
+# kh 21.02.22 orig (disabled for a special test)
```

Comment:
This still needs a general solution before it goes into the main branch. I would for the `AWIESM` side recommend something similar to `use_cvmix` as we already have for other "optional" features. However, `cvmix` capability depends on the version of `FESOM` being used (I think??), and I am not yet sure if the one we give with `AWIESM 2.1` has that.

Reply:
It seems the version given to you with `awiesm-2.1` does indeed include cvmix. https://github.com/FESOM/fesom2/tree/2.1-awiesm-2.1/src

Reply:
AWIESM uses `fesom-2.1.yaml`, not `fesom-2.0.yaml`. I'll get rid of these comments.